codec_test.go

```diff
@@ -22,11 +22,12 @@ func TestCodec_String(t *testing.T) {
 }
 
 func BenchmarkEncoding(b *testing.B) {
+	fmt.Println(benchmarkDivider)
```

Comment:
IMO, this divider is not necessary. Also, I'm not sure if tools like benchstat can deal with that.

Reply:
The divider is there for the other benchmarks, so I just made everything even. Do we want to remove all the dividers? I'm impartial.

Reply:
Yeah, let's remove all of them.

Reply:
Removed in 3910e3b.